site/app/views/admin/PlagiarismView.php

```diff
@@ -88,7 +88,8 @@ public function plagiarismMainPage($semester, $course, $gradeables_with_plagiari
     }
 
     public function showPlagiarismResult($semester, $course, $gradeable_id, $gradeable_title, $rankings) {
-        $this->core->getOutput()->addBreadcrumb('Plagiarism Detection', $this->core->buildCourseUrl(['plagiarism']));
+        $this->core->getOutput()->addBreadcrumb('Plagiarism', $this->core->buildCourseUrl(['plagiarism']));
+        $this->core->getOutput()->addBreadcrumb($gradeable_title, $this->core->buildCourseUrl([$gradeable_title]));
```

Comment:
I believe links are only added to all breadcrumbs except the last most since that indicates the page you're currently on so you can removed the `$this->core->buildCourseUrl([$gradeable_title])` on ln 92 since it doesn't add a link:

<img width="887" alt="image" src="https://user-images.githubusercontent.com/12129065/119135496-804f8100-ba0c-11eb-8dcc-866491d76aed.png">

Reply:
How is it any different from before though?  It seemed to me as if you added things to the breadcrumb and then the breadcrumb generation engine linkified all but the last one.  If I removed the gradeable name, I believe that it would stop at "Plagiarism Detection" with no link and the user would be unable to return to the previous page.

Reply:
I mean just the build course url part

Reply:
Oh, I see.  I thought that was necessary to link it in case there was a breadcrumb path even further down.  I'll remove it.